src/components/layout/footer/disclaimer.js

```diff
@@ -9,7 +10,7 @@ import {
 import { Show } from 'components/containers'
 import { Localize, localize } from 'components/localization'
 
-const DisclaimerSection = () => {
+const DisclaimerSection = ({ academy }) => {
```

Comment:
Better to name this `is_academy` to reflect it's indeed a boolean.

Reply:
yes agree, thank u salar for your revision :))))